recipes/single-source/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "single-source" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1bdbf77cacd01849dd299b9a24bb550a15f60803ca42339748d3c3b4c7de7766
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
```

Comment:
It looks like you need `poetry` as a `host` requirement.

Reply:
@synapticarbors yes, it needs. It was late at night =)